scrapy/core/downloader/tls.py

```diff
@@ -65,7 +73,7 @@ def _identityVerifyingInfoCallback(self, connection, where, ret):
             elif where & SSL_CB_HANDSHAKE_DONE:
                 try:
                     verifyHostname(connection, self._hostnameASCII)
-                except VerificationError as e:
+                except verification_errors as e:
```

Comment:
Could you please update docstring of ScrapyClientTLSOptions? It mentions VerificationError and ValueError, but not CertificateError.

Reply:
+1 & done